api/src/main/resources/workbench.yaml

```diff
@@ -1165,6 +1165,25 @@ paths:
           schema:
             $ref: '#/definitions/ErrorResponse'
 
+  /v1/cron/billingProjectBufferStatus:
```

Comment:
This sounds like a can of worms for another day. I'm thinking of possibly having ops- or monitoring-specific roots soon under one of Scout's projects, which may be a better home for this one.

Reply:
Yeah, this is more about monitoring than operations, unless we need to expose a methods to evict projects form the buffer or something.

Also, I'm wondering whether it would make sense to create a new Authority for devops, separate from what support needs. I.e. we don't need to adjust individual users in most cases, but there are things we may need to do that support or other roles needn't have access to. Does the authority system currently assume a strictly totem-pole hierarchy, or can we have mutually exclusive privileges?

Reply:
The latter: https://github.com/all-of-us/workbench/blob/048a8a0744f7421352f315fc8ff0d0f70e0eecb9/api/src/main/resources/workbench.yaml#L2262-L2271

There's an argument that the UI might want to call this to show some UX on the workspace page, so I wouldn't bother with an authority for now unless a broader logical grouping emerges.

Reply:
Sounds good for now, thanks. Was just thinking down the road if we start adding more specialized endpoints.